plugins/gimp/file-jxl-save.cc

```diff
@@ -194,69 +186,99 @@ bool SaveDialog() {
   gtk_widget_show(main_vbox);
 
   // Standard Settings Frame
-  frame = gtk_frame_new("Standard Settings");
+  frame = gtk_frame_new(nullptr);
   gtk_frame_set_shadow_type(GTK_FRAME(frame), GTK_SHADOW_ETCHED_IN);
-  gtk_box_pack_start(GTK_BOX(main_vbox), frame, true, true, 0);
+  gtk_box_pack_start(GTK_BOX(main_vbox), frame, false, false, 0);
   gtk_widget_show(frame);
 
   vbox = gtk_vbox_new(false, 6);
   gtk_container_set_border_width(GTK_CONTAINER(vbox), 6);
   gtk_container_add(GTK_CONTAINER(frame), vbox);
   gtk_widget_show(vbox);
 
-  // Butteraugli Distance
-  static gchar distance_help[] =
-      "Butteraugli distance.  Use lower values for higher quality.";
-  frame = gtk_frame_new("Butteraugli Distance");
-  gimp_help_set_help_data(frame, distance_help, nullptr);
-  gtk_frame_set_shadow_type(GTK_FRAME(frame), GTK_SHADOW_NONE);
-  gtk_box_pack_start(GTK_BOX(vbox), frame, false, false, 0);
-  gtk_widget_show(frame);
-
-  // Distance Scale
-  table = gtk_table_new(1, 3, false);
+  // Distance / Quality Table
+  table = gtk_table_new(20, 3, false);
   gtk_table_set_col_spacings(GTK_TABLE(table), 6);
-  gtk_container_add(GTK_CONTAINER(frame), table);
+  gtk_box_pack_start(GTK_BOX(vbox), table, false, false, 0);
   gtk_widget_show(table);
 
+  static gchar distance_help[] =
+      "Butteraugli distance.  Use lower values for higher quality.";
```

Comment:
Technically, it's a distance _target_, and the encoder doesn't necessarily hit the target (especially at low effort).

Reply:
How about:
```
Butteraugli distance target.  Lower values indicate higher quality.
   d = 0.3   Excellent
   d = 1     Very Good
   d = 2     Good
   d = 3     Fair
   d = 6     Poor
```

Reply:
Yes, something like that makes sense to me. Of course such mappings from numbers to words are always a bit subjective and use case dependent.

Reply:
"Butteraugli distance target.  Suggested starting values..."